Webapp/sources/services/data_export_history.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import asdict, dataclass
+
+from flask import current_app, json
+from sources.services.message_queue import MessageSerdeMixin
+
+
+@dataclass
+class DataExportMessage(MessageSerdeMixin):
+    """Class for creating a kafka message for the data_export_history topic"""
+
+    person: int
+    workgroup: int
+    workbook: int
+    reactions: list[int]
+    date: str
+
+    def serialise(self):
+        """Convert a message into a JSON object with a schema and payload.
+
+        The schema is an object that lists the fields and their types.
+        The payload is an object representing the message class in JSON format.
+
+        Returns:
+            dict: The message class formatted with schema and payload.
+        """
+        schema = {
+            "type": "struct",
+            "optional": False,
+            "fields": [
+                {"field": "person", "type": "int32"},
+                {"field": "workgroup", "type": "int32"},
+                {"field": "workbook", "type": "int32"},
+                {"field": "reactions", "type": "list"},
+                {"field": "date", "type": "string"},
+            ],
+        }
+        payload = asdict(self)
```

Comment:
In the new audit log system, you will not need the schema anymore. I think you should be fine to return the serialised payload alone